### ISSUE TITLE
Arbiter signals unmanaged configuration

### ISSUE THREAD

**Issue by mohierf** [Alignak-monitoring/alignak#822]
Sometimes the arbiter log that the some daemons do not manage their configuration, with such logs:
```
[2017-05-22 06:04:23 UTC] WARNING: [alignak.dispatcher] Scheduler 'scheduler-master' do not manage this configuration: eadebf89c38e46a49fe646905d66d899, I am not happy.
[2017-05-22 06:04:23 UTC] WARNING: [alignak.dispatcher] [All] The reactionner reactionner-master manage a unmanaged configuration
[2017-05-22 06:04:23 UTC] WARNING: [alignak.dispatcher] [All] The poller poller-master manage a unmanaged configuration
[2017-05-22 06:04:23 UTC] WARNING: [alignak.dispatcher] [All] The broker broker-master manage a unmanaged configuration
[2017-05-22 06:04:23 UTC] WARNING: [alignak.dispatcher] [All] The receiver receiver-master manage a unmanaged configuration
```

What is happenning? I am currenlty investigating on this problem which is due to timeout occuring in the communication between the daemons ...

**Comment by mohierf** [comment]
Looks better as of now, but keep this issue opened for tracking

**Comment by mohierf** [comment]
Closed 2.0
